x-pack/plugins/cases/public/components/all_cases/use_cases_columns_selection.test.tsx

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { licensingMock } from '@kbn/licensing-plugin/public/mocks';
+import { renderHook } from '@testing-library/react-hooks';
+
+import type { AppMockRenderer } from '../../common/mock';
+
+import { createAppMockRenderer } from '../../common/mock';
+import { useCasesColumnsSelection } from './use_cases_columns_selection';
+import { mergeSelectedColumnsWithConfiguration } from './utils';
+import { useCasesColumnsConfiguration } from './use_cases_columns_configuration';
+import { DEFAULT_CASES_TABLE_COLUMNS } from '../../../common/constants';
+
+jest.mock('./use_cases_columns_configuration');
+jest.mock('./utils');
```

Comment:
I don't think we should mock the `utils`. There is no API calls or similar to justify it.